tik/Util.cpp

```diff
@@ -14,6 +15,15 @@ void PrintVal(llvm::Value *val)
     std::cout << str << "\n";
 }
 
+void PrintVal(llvm::Module *mod)
+{
+    AssemblyAnnotationWriter *write = new llvm::AssemblyAnnotationWriter();
+    std::string str;
+    llvm::raw_string_ostream rso(str);
+    mod->print(rso, write);
+    std::cout << str;
```

Comment:
I think this needs a new line at the end.

Reply:
done